core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -19,6 +19,98 @@
+  var translationWelcomeModal = element(
+    by.css('.protractor-test-welcome-modal'));
+
+  /*
+   Actions
+  */
+
+  // TUTORIAL
+
+  this.exitTutorial = function() {
+    // If the translate welcome modal shows up, exit it.
+    translationWelcomeModal.isPresent().then(function(isVisible) {
+      if (isVisible) {
+        waitFor.elementToBeClickable(
+          dismissWelcomeModalButton,
+          'Welcome modal is taking too long to appear');
+        dismissWelcomeModalButton.click();
+      }
+    });
+
+    waitFor.invisibilityOf(
+      translationWelcomeModal,
+      'Translate Welcome modal takes too long to disappear');
+
+    // Otherwise, if the editor tutorial shows up, exit it.
+    element.all(by.css('.skipBtn')).then(function(buttons) {
+      if (buttons.length === 1) {
+        buttons[0].click();
+      } else if (buttons.length !== 0) {
+        throw 'Expected to find at most one \'exit tutorial\' button';
+      }
+    });
+  };
+
+  this.finishTutorial = function() {
+    // Finish the tutorial.
+    var finishTutorialButton = element.all(by.buttonText('Finish'));
+    waitFor.elementToBeClickable(
+      finishTutorialButton.first(),
+      'Finish Tutorial Stage button is not clickable');
+    finishTutorialButton.then(function(buttons) {
+      if (buttons.length === 1) {
+        buttons[0].click();
+      } else {
+        throw Error('There is more than 1 Finish button!');
+      }
+    });
+  };
+  this.playTutorial = function() {
+    var tutorialTabHeadings = [
+      'Translations In Oppia',
+      'Choose Language',
+      'Choose a Card to Translate',
+      'Choose a Part of the Card to Translate',
+      'Recording Audio',
+      'Re-record/Re-upload audio'
+    ];
+    tutorialTabHeadings.forEach(function(heading) {
+      var tutorialTabHeadingElement = element(by.cssContainingText(
+        '.popover-title', heading));
+      waitFor.visibilityOf(
+        tutorialTabHeadingElement, 'Tutorial: ' + heading + 'is not visible');
+      // Progress to the next instruction in the tutorial.
+      var nextTutorialStageButton = element.all(by.css('.nextBtn'));
+      waitFor.elementToBeClickable(
+        nextTutorialStageButton.first(),
+        'Next Tutorial Stage button is not clickable');
+      nextTutorialStageButton.then(function(buttons) {
+        if (buttons.length === 1) {
+          buttons[0].click();
+          waitFor.invisibilityOf(
+            tutorialTabHeadingElement,
+            'Tutorial stage takes too long to disappear');
+        } else {
+          throw Error('There is more than one Next button!');
+        }
+      });
+    });
+  };
+
+  this.startTutorial = function() {
+    waitFor.visibilityOf(
+      translationWelcomeModal,
+      'Translate Welcome modal takes too long to appear');
+    element(by.css('.protractor-test-start-tutorial')).click();
+    waitFor.visibilityOf(
+      element(by.css('.ng-joyride-title')),
+      'Tutorial modal takes too long to appear');
```

Comment:
Can we make this message similar to other places?

Reply:
So make this 'Translation tutorial modal takes too long to appear.'.

Reply:
Yep, sounds good!

Reply:
Changed it, so it's done.